book/src/tutorial_with_flutter.md

```diff
@@ -27,16 +28,7 @@ git clone https://github.com/fzyzcjy/flutter_rust_bridge && cd flutter_rust_brid
 
 This step is optional, since I have generated the source code already (in quickstart). Even if you do it, you should not see anything changed.
 
-Firstly, you may need to install dependencies of this library:
-
-* Install `ffigen`:  `dart pub global activate ffigen`, and [install LLVM](https://pub.dev/packages/ffigen#installing-llvm) which is required by `ffigen`.
-* Install the codegen program: `cargo install flutter_rust_bridge_codegen` (or other probably faster methods described [here](quickstart.md)).
-
-Then, run the code generator:
-
-```shell
-flutter_rust_bridge_codegen --rust-input frb_example/with_flutter/rust/src/api.rs --dart-output frb_example/with_flutter/lib/bridge_generated.dart --c-output frb_example/with_flutter/ios/Runner/bridge_generated.h
-```
+As soon as you make any modification to `api.rs`, you need to run codegen again : more informations about requirements for code generation in the [Installing dependencies](integrate/deps.md) section.
```

Comment:
uber-nit about text

```suggestion
As soon as you make any modification to `api.rs`, you need to run codegen again. More informations about requirements for code generation can be seen in the [Installing dependencies](integrate/deps.md) section.
```

Reply:
done in ef86e5bce66da7565fdc86e2301efd2983d9a974